app/home/page.js

```diff
@@ -0,0 +1,44 @@
+"use client";
+import React, {useState, useEffect} from 'react';
+
+  async function fetchData() {
+    const response = await fetch('https://api.openweathermap.org/data/2.5/weather?q=Mar%20del%20Plata,AR&appid=a3f3933bf95b00578ff3adc1d8402640&units=metric');
+    
+    if (response.ok) {
+      const data = await response.json();
+      return data;
+    } else {
+      throw new Error('Error en encontrar data');
+    }
+  }
+  
+  export default function Page() {
```

Comment:
Podemos usar un nombre mas puntual para este componente:

```suggestion
  export default function Temperatura() {
```